content/knowledge-codemagic/build-versioning.md

```diff
@@ -314,3 +314,116 @@ Provided you have exported your Google Play Console service account credentials
 
 
 Alternatively, you can add a custom [Pre-build script](/flutter-configuration/custom-scripts) and write the build number to a file, which will be read from your `android/app/build.gradle` during the build (See details [here](https://github.com/codemagic-ci-cd/android-versioning-example/tree/autoversioning_through_file)).
+
+
+
+## Firebase latest build version
+
+Use [`firebase-app-distribution get-latest-build-version`](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/firebase-app-distribution/get-latest-build-version.md#get-latest-build-version) action from Codemagic CLI tools to get the latest build version from Firebase.
```

Comment:
Firebase has many services; here too I would prefer to explicitly mention Firebase App Distribution